#request
string state
---
#response
bool alive
